.editorconfig

```diff
@@ -0,0 +1,9 @@
+root = true
+
+[*]
+end_of_line = lf
+indent_style = space
+indent_size = 4
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
```

Comment:
Would you mind putting the directives in alphabetical order? Also, putting the line-length in here will usually cause compliant text editors to show a visual bar in the interface, so you can see where the length limit is. Example configuration: https://github.com/justinmayer/kagi/blob/master/.editorconfig